README.adoc

```diff
@@ -212,19 +297,80 @@ IMPORTANT: This command need to be executed from the same directory where the in
 
 NOTE: See more over it in the Readme of https://github.com/matryer/moq[moq]
 
-== Developing Helpers
-=== Errcheck
+* Using the mock
 
-The https://github.com/kisielk/errcheck[errcheck] is required to run the `make errcheck` command.
+In the created file you will see an implementation commented as in the following example. This implementation will be used in the test file to mock the methods/func of this interface.
 
-Install:
+[source,go]
+----
+// AppServiceMock is a mock implementation of Service.
+//
+//     func TestSomethingThatUsesAppService(t *testing.T) {
+//
+//         // make and configure a mocked Service
+//         mockedAppService := &AppServiceMock{
+//             GetAppsFunc: func() (*[]models.App, error) {
+// 	               panic("mock out the GetApps method")
+//             },
+//         }
+//
+//         // use mockedAppService in code that requires Service
+//         // and then make assertions.
+//
+//     }
+----
 
-[source,shell]
+* Adding data to the mock function
```

Comment:
the word 'mock' works well here, but in other headings it's not so good, eg 'Editing the mock'. 
@austincunningham  can we avoid using an adjective as a noun, even if we use 'mock system' or similar?